src/main/java/core/basesyntax/Lottery.java

```diff
@@ -3,12 +3,13 @@
 import java.util.Random;
 
 public class Lottery {
-    public static final int RANDOM = 101;
+    public static final int MAX_BALL_NUMBER = 101;
 
     private Random random = new Random();
     private ColorSupplier colorSupplier = new ColorSupplier();
 
     public Ball getRandomBall() {
-        return new Ball(random.nextInt(RANDOM), colorSupplier.getRandomColor());
+        return new Ball(random.nextInt(MAX_BALL_NUMBER), colorSupplier.getRandomColor());
     }
 }
+
```

Comment:
this empty line here is redundant, we should have only 1 at the end of the file
Hint: check it in the IDE because github shows it a bit different 😉